Frameworks/CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -746,6 +746,12 @@ static CFRange __CFCalendarGetRangeOfUnit2(CFCalendarRef calendar, CFCalendarUni
     // WINOBJC: Add cases for weekOf units. ICU will handle them below.
     case kCFCalendarUnitWeekOfYear:
     case kCFCalendarUnitWeekOfMonth:
+    // WINOBJC: Asking for the range of WeekOfMonth in Year always returns 6.
```

Comment:
It begs the question as to why other units are fast tracked to return constant values here though.  Everything from second up through hour seems incorrect to then return constant values. Especially given that there hopefully aren't 60 seconds in a year. These should also be addressed in the work done for fixing calculate.

I think the workaround to return a constant 6 is sufficient to match the reference platform on this one case for the time being. I've yet to find a case where it's incorrect even if it's a faulty mechanism for determining the actual values. Especially given the other short cuts that core foundation has already taken for those smaller values I think it's acceptable until the actual calculations are fixed.

---
In reply to: [74473193](https://github.com/Microsoft/WinObjC/pull/844#discussion_r74473193) [](ancestors = 74473193,74468177,74461625,74359393,74356593,74354002,74342094,74336418)